config/v1/types_cluster_version.go

```diff
@@ -95,6 +102,12 @@ type ClusterVersionStatus struct {
 	// +required
 	Desired Release `json:"desired"`
 
+	// architecture identifies the architecture of the cluster nodes. If
+	// the cluster architecture is heterogeneous, whereby the cluster
+	// has nodes running on different architectures, "multi" is used.
+	// +optional
```

Comment:
explain whether empty is a temporary condition or a long term condition to help consumers know what to expect.